dashboard/app/views/home/index.html.haml

```diff
@@ -1,3 +1,5 @@
+= render partial: 'layouts/survey'
```

Comment:
I think changing this to `= render partial: 'layouts/survey' if current_user` will fix your test failures